### PR TITLE
Add `MLKitQrCodeAnalyzer`

### PR DESCRIPTION
**Pull request by msasikanth**

### CODE REVIEW

File: app/src/main/java/org/simple/clinic/widgets/qrcodescanner/MLKitQrCodeAnalyzer.kt [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  private val scanner = BarcodeScanning.getClient(options)
+
+
+  @SuppressLint("UnsafeExperimentalUsageError")
+  override fun analyze(imageProxy: ImageProxy) {
+    val mediaImage = imageProxy.image
+    if (mediaImage != null) {
+      val image = InputImage.fromMediaImage(mediaImage, imageProxy.imageInfo.rotationDegrees)
+      scanner.process(image)
+          .addOnSuccessListener { barcodes ->
```
Comment (janhavisinghh) [inline app/src/main/java/org/simple/clinic/widgets/qrcodescanner/MLKitQrCodeAnalyzer.kt:27]: "Each Barcode object represents a barcode that was detected in the image." Does this mean that the list of barcodes is the multiple images that are taken while scanning?

Comment (msasikanth) [reply app/src/main/java/org/simple/clinic/widgets/qrcodescanner/MLKitQrCodeAnalyzer.kt:27]: No, it represents multiple barcodes detected on a single scan.

**Comment by janhavisinghh** [comment]
Can you please fix the code climate issue?

**Review by janhavisinghh — APPROVED** [review]
LGTM

**Review by janhavisinghh — APPROVED** [review]

**Pull request merged**